src/number.cr

```diff
@@ -161,9 +161,10 @@ struct Number
     {(self / number).floor, self % number}
   end
 
-  # Implements the comparison operator.
+  # The comparison operator.
   #
-  # See also: `Object#<=>`.
+  # Returns `-1` if `self` is less than *other*, `0` if `self` equals *other*
```

Comment:
IMO readability would be improved if this followed the schema used in the other descriptions to first name the different types of outcome and then the reasons for each.